requirements/edx/base.txt

```diff
@@ -192,7 +192,7 @@ pygments==2.4.2
 pygraphviz==1.5
 pyjwkest==1.3.2
 pyjwt==1.5.2
-pymongo==2.9.1
+pymongo==3.8.0
```

Comment:
`pymongo==3.9.0` has been released, do you think it would be hard to upgrade?
https://api.mongodb.com/python/current/changelog.html#changes-in-version-3-9-0

Reply:
Done. Note that `3.9.0` might not have an available version when backporting to older releases.